src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,56 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
-);
+export const MovieCard = ({ movie }) => {
+  const {
+    imgUrl,
+    title,
+    description,
+    imdbUrl,
+  } = movie;
+
+  return (
+    <div
```

Comment:
I am not sure whether it is necessary to move the div attributes to the next row, we should do it when have more than 3 attributes. But its readable and looks good.